### PR TITLE
Replace MD templates with YAML

### PR DESCRIPTION
**Pull request by AdamMeza-Bentley**
## Changes

- updated the issue templates from md to yaml at the request of @FlyersPh9 

## Testing

N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Ask a question
-about: Please use the community forum (https://github.com/iTwin/appui/issues) for general questions about using iTwin.js AppUI.
+about: Please use the community forum(https://github.com/iTwin/appui/issues) for general questions about using iTwin.js AppUI.
```
Comment (GerardasB) [inline .github/ISSUE_TEMPLATE/question.md:3]: Unintentional edit?
Could we replace this with https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser `contact_links` and direct to our `https://github.com/iTwin/appui/discussions` page instead?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      description: If applicable, add screenshots to help explain your problem.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Desktop (please complete the applicable information)
+      placeholder: |
+        - OS: [e.g. iOS]
+        - Browser [e.g. chrome, safari]
+        - Version [e.g. 22]
```
Comment (GerardasB) [inline .github/ISSUE_TEMPLATE/bug_report.yml:46]:
```suggestion
        - Browser Version [e.g. 22]
```
Should we specify that it is a browser version?

**Review by GerardasB — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (AdamMeza-Bentley) [reply .github/ISSUE_TEMPLATE/bug_report.yml:46]: I can make that edit no problem!

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (AdamMeza-Bentley) [reply .github/ISSUE_TEMPLATE/question.md:3]: To clarify when you say "replace this" you mean updating the `config` to the template you provided and moving the `about` value to that file?

Comment (AdamMeza-Bentley) [reply .github/ISSUE_TEMPLATE/question.md:3]: also yes that edit was unintentional

Comment (GerardasB) [reply .github/ISSUE_TEMPLATE/question.md:3]: Yes, let's get rid of this template and add the about section to the `config.yml` :)

**Pull request merged**